sdk/tests/conftest.py

```diff
@@ -223,17 +223,38 @@ def generic_kafka_config():
     )
 
 
-@pytest.fixture
-def log_config(generic_log_level, generic_result_logger_type, generic_bigquery_config, generic_kafka_config):
-    return turing.generated.models.RouterVersionLogConfig(
-        log_level=generic_log_level,
-        custom_metrics_enabled=True,
-        fiber_debug_log_enabled=True,
-        jaeger_enabled=True,
-        result_logger_type=generic_result_logger_type,
-        bigquery_config=generic_bigquery_config,
-        kafka_config=generic_kafka_config
-    )
+@pytest.fixture(params=["kafka", "bigquery", "others"])
+def log_config(generic_log_level, generic_result_logger_type, generic_bigquery_config, generic_kafka_config, request):
+    if request.param == "kafka":
```

Comment:
One trick you can do to remove the verbosity is something like this:
```python
params = dict(
    log_level=generic_log_level,
    custom_metrics_enabled=True,
    fiber_debug_log_enabled=True,
    jaeger_enabled=True,
    result_logger_type=turing.generated.models.ResultLoggerType("kafka"),
    bigquery_config=None,
    kafka_config=generic_kafka_config,
)
if request.param == "bigquery":
    params = {result_logger_type: turing.generated.models.ResultLoggerType("kafka"), **params}
...
return turing.generated.models.RouterVersionLogConfig(**params)
```

Reply:
Ok I've refactored it slightly differently so as to ensure `kafka_config` or `bigquery_config` gets properly assigned when their corresponding `result_logger_type` is set:

```python
result_logger_type = generic_result_logger_type.value if request.param == "others" else request.param

params = dict(
    log_level=generic_log_level,
    custom_metrics_enabled=True,
    fiber_debug_log_enabled=True,
    jaeger_enabled=True,
    result_logger_type=turing.generated.models.ResultLoggerType(result_logger_type),
    bigquery_config=None,
    kafka_config=None
)

if request.param == "kafka":
    params["kafka_config"] = generic_kafka_config
elif request.param == "biggquery":
    params["bigquery_config"] = generic_bigquery_config

return turing.generated.models.RouterVersionLogConfig(**params)
```